docs/components/select.md

```diff
@@ -231,6 +231,80 @@ KSelect works as regular inputs do using v-model for data binding:
 </Komponent>
 ```
 
+### autosuggest
+
+With `autosuggest`, you can type in the input, trigger a query to an API with the search keyword, and then update `items` asynchronously as suggestions.
+Loading state and empty state can be configured using the `loading` and `empty` slots.
+
+<KSelect
+  autosuggest
+  :items="itemsForAutosuggest"
+  :loading="loading"
+  appearance="select"
+  @keyword-change="onKeywordChange"
+  @focus="onFocus"
+>
+  <template v-slot:loading>
+    <div>Loading...</div>
+  </template>
+  <template v-slot:empty>
+    <div>No results found</div>
+  </template>
+</KSelect>
```

Comment:
Most of the time in Konnect, we'll be using the slotted syntax with the `autosuggest` prop. Can we update this example to utilize the slotted syntax?

You will also need to add a new `item.description` to the data object. I added suggestions in those places.

One other thing, you could update your filter function to check the `label` and the `description` for autosuggest, if you would like.

```suggestion
<KSelect
  autosuggest
  :items="itemsForAutosuggest"
  :loading="loading"
  appearance="select"
  @keyword-change="onKeywordChange"
  @focus="onFocus"
>
  <template v-slot:item-template="{ item }">
    <div class="select-item-label">{{ item.label }}</div>
    <div class="select-item-desc">{{ item.description }}</div>
  </template>
  <template v-slot:loading>
    <div>Loading...</div>
  </template>
  <template v-slot:empty>
    <div>No results found</div>
  </template>
</KSelect>
```

Reply:
OK, now the filter function checks both `label` and `description`.